### ISSUE TITLE
Saw your challengepost submission

### ISSUE THREAD

**Issue by ardent-android** [arya2/Julius#7]
Just curious, so the program detects if someone is suffering a seizure through the webcam? And isn't it usually 3-30 fps per second that people with epilepsy immediately get a seizure? Nice project btw! :+1:

**Comment by arya2** [comment]
Nah, we couldn't figure out native screen capture and used a program that fed screen capture in as a mock cam source
